tests/test_http_client_injection.py

```
@@ -0,0 +1,190 @@
+        # QdrantClient is closed.
+        injected = httpx.Client()
+        client = QdrantClient(check_compatibility=False, http_client=injected)
+        client.close()  # should be a no-op for the injected client
+        try:
+            # The injected client is still usable: dispatch through it. Any
+            # transport error is fine; the point is that the client itself
+            # is still open and method dispatch works (i.e. it wasn't
+            # silently closed by QdrantClient.close()).
+            injected.get("http://localhost:1/nope")
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)